src/js/bootstrap-slider.js

```diff
@@ -372,6 +372,9 @@ const windowIsDefined = (typeof window === "object");
 			options = options ? options : {};
 			var optionTypes = Object.keys(this.defaultOptions);
 
+			var isMinSet = 'min' in options;
```

Comment:
could you refactor this to use `hasOwnProperty` instead of the `in` keyword?  `hasOwnProperty` checks the object only versus traversing the prototype chain

Reply:
also, refactor these `var`s to be `const`s (since their values do not change)